src/Dhall/Pretty/Internal.hs

```diff
@@ -1084,6 +1094,10 @@ buildExprF DoubleShow =
     "Double/show"
 buildExprF Text =
     "Text"
+buildExprF FilePath =
+    "FilePath"
+buildExprF Url =
+    "Url"
```

Comment:
Perhaps `Url` should be capitalized as `URL` since it is an acronym

Reply:
Oh, sorry, I always capitalise this way, as most coding guidelines suggest using camel case with acronyms, so I didn’t even think about it.

Anyway, I can capitalise the text representation of the type, but if I change the constructor itself, then it will conflict with `URL` from `ImportType`. I guess, I can add `Import` as a prefix to constructors of `ImportType`. What do you think?